src/static/js/media.js

```diff
@@ -961,8 +961,8 @@ $(function() {
                     name: defaultname,
                     email: defaultemail,
                     toaddresses: toaddresses,
-                    subject: common.replace_all(media.selected_filenames(), ".html", ".pdf"),
-                    attachments: common.replace_all(media.selected_filenames(), ".html", ".pdf"),
+                    subject: media.selected_filenames(),
+                    attachments: media.selected_filenames(),
```

Comment:
I think we still need to do the replace here - this is for the attachment filenames shown in the send email dialog, and we need to indicate to the user that any html files will be pdfs by the time the recipient gets them